Demo/Tests/ReactiveCocoaMoyaProviderTests.swift

```diff
@@ -60,10 +60,10 @@ class ReactiveCocoaMoyaProviderSpec: QuickSpec {
             }
 
             class TestProvider<Target: TargetType>: ReactiveCocoaMoyaProvider<Target> {
-                init(endpointClosure: @escaping EndpointClosure = MoyaProvider.DefaultEndpointMapping,
-                    requestClosure: @escaping RequestClosure = MoyaProvider.DefaultRequestMapping,
-                    stubClosure: @escaping StubClosure = MoyaProvider.NeverStub,
-                    manager: Manager = MoyaProvider<Target>.DefaultAlamofireManager(),
+                init(endpointClosure: @escaping EndpointClosure = MoyaProvider.defaultEndpointMapping,
+                    requestClosure: @escaping RequestClosure = MoyaProvider.defaultRequestMapping,
+                    stubClosure: @escaping StubClosure = MoyaProvider.neverStub,
+                    manager: Manager = MoyaProvider<Target>.defaultAlamofireManager(),
```

Comment:
For context, this was originally written as a function, since it returns a new manager instance every time (rather than an identical closure). We can totally revisit that though.